code/__DEFINES/dcs/signals/signals_revolution.dm

```diff
@@ -0,0 +1,4 @@
+/// Revolution signals
+
+///Called when the crew wins the revolution onto machines with the payment component associated with the LATHE_TAX define. No arguments.
+#define COMSIG_REVOLUTION_TAX_REMOVAL "revs_tax_removal"
```

Comment:
move this to global_signals.dm or whatever it's called, prefix its string with !, and make sure GLOB is in the define so it's clear what it is